rango/views.py

```diff
@@ -3,8 +3,21 @@
 
 
 def index(request):
-    return HttpResponse("Rango says hey there partner! <a href='/rango/about/'>About</a>")
+    # Construct a dictionary to pass to the template engine as its context.
+    # Note the key boldmessage matches to {{ boldmessage }} in the template!
+    context_dict = {'boldmessage': 'Crunchy, creamy, cookie, candy, cupcake!'}
+
+    # Return a rendered response to send to the client.
+    # We make use of the shortcut function to make our lives easier.
+    # Note that the first parameter is the template we wish to use.
+    return render(request, 'rango/index.html', context=context_dict)
 
 
 def about(request):
-    return HttpResponse("Rango says here is the about page. <a href='/rango/'>Index</a>")
+    context_dict = {'message': 'Funny man!'}
+    return render(request, 'rango/about.html', context=context_dict)
+
+
+def training(request):
+    example_dict = {'variable_name': 'Below presented to Rango and his Hot Baby. Let`s scroll together!'}
```

Comment:
'variable_name' is wrong naming, as I could not understand what data you want to pass in this variable. 
Please, change the naming.